assets/src/edit-story/app/highlights/quickActions/useQuickActions.js

```diff
@@ -231,14 +234,8 @@ const useQuickActions = () => {
     handleMouseDown,
   ]);
 
-  const foregroundImageActions = useMemo(
+  const foregroundCommonActions = useMemo(
```

Comment:
:smart:

Reply:
nice idea! 

let's keep an eye on this for ordering discontinuity.